components/spinner/test/index.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+/**
+ * Internal dependencies
+ */
+import Spinner from '../';
+
+describe( 'Spinner', () => {
+	describe( 'basic rendering', () => {
+		it( 'should match the following element', () => {
+			const element = <span className="spinner is-active" />;
+			const spinner = shallow( <Spinner /> );
+			expect( spinner.matchesElement( element ) ).to.be.true();
```

Comment:
I have found tests like this useful when I change markup in a component and forget to update the styles.  When things fail it reminds me to update the style.  I can understand them not being useful persee, but I typically aim for 100% coverage.  On projects that move relatively fast like this, I think more tests over fewer tests is better. It should probably be decided what we are going to test, how, and why.